app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkItemMenu.kt

```diff
@@ -25,14 +26,20 @@ class BookmarkItemMenu(
         Share,
         OpenInNewTab,
         OpenInPrivateTab,
+        OpenAllInTabs,
+        OpenAllInPrivateTabs,
         Delete,
         ;
     }
 
     val menuController: MenuController by lazy { BrowserMenuController() }
 
     @VisibleForTesting
-    internal fun menuItems(itemType: BookmarkNodeType): List<TextMenuCandidate> {
+    @SuppressWarnings("LongMethod")
+    internal suspend fun menuItems(itemType: BookmarkNodeType, itemId: String): List<TextMenuCandidate> {
+        // if have at least one child
```

Comment:
This is an example where I really don't think the comment is adding much for us. So, I would ask that we audit the comments and try to add clarity to them in the PR.